forte/data/caster.py

```diff
@@ -78,3 +81,46 @@ def input_pack_type():
     @staticmethod
     def output_pack_type():
         return MultiPack
+
+
+class DataPackBoxer(Caster[MultiPack, DataPack]):
+    """
+    This class creates a DataPack from a MultiPack, this DataPack is the only
+    content of the original MultiPack.
+    """
+
+    def cast(self, pack: MultiPack) -> DataPack:
+        """
+        Auto-box the MultiPack into a DataPack by using pack_index to take the unique pack.
+
+        Args:
+            pack: The MultiPack to be boxed.
+
+        Returns: A DataPack boxed from the MultiPack.
+
+        """
+
+        p = pack.get_pack_at(0)
+
+        if p.pack_name:
+            self.configs.pack_name = (
+                p.pack_name.strip("_multi")
+                if "_multi" in p.pack_name
+                else p.pack_name
+            )
+        else:
+            self.configs.pack_name = None
```

Comment:
What is this part doing here? Looks like they change the values in `self.configs`. But why do we need to change the config? They will not affect anything useful to the users.

What we need to do here, is to return the pack from the multipack that matches the `pack_name` or `pack_index`.